src/modules/UI/components/HelpModal/style.js

```diff
@@ -23,7 +22,7 @@ export default StyleSheet.create({
     padding: 4
   },
   modalMiddleWebView: {
-
+    height: 240
```

Comment:
This shouldn't be hardcoded but instead stretch to nearly fill the full screen. Ideally only about 10dp margin from top and bottom. Same for sides.